src/main/java/dev/dokan/dokan_java/Unsigned.java

```diff
@@ -0,0 +1,17 @@
+package dev.dokan.dokan_java;
+
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+
+
+@Documented
```

Comment:
Please add some explaining JavaDoc. The following questions should be answered:

1. What does this annotation mean.
1. Why is this annotation needed.
1. Are there pitfalls (e.g. division of unsigned numbers)

Reply:
Revolved by commit b9aa49a902581c5ad01f01aeed07096d889973b0